File: src/main/java/com/train/Tester.java
package com.train;

import java.util.Scanner;

public class Tester {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int totalt = 0;
        int roundt = 0;
        do {System.out.println("Please enter number of tickets: ");
            totalt = scanner.nextInt();
            if(totalt==-1){
                System.out.println("End");break;}
            else { System.out.println("How many round-trip tickets: ");
                    roundt = scanner.nextInt();}
                    Ticket ticket = new Ticket(totalt, roundt);
                    ticket.print();
        }while(totalt!=-1);

    }
}

File: src/main/kotlin/com/train/Tester.kt
package com.train

import java.util.*

fun main(){
    val scanner = Scanner(System.`in`)
    var totalt = 0;
    var roundt = 0;
    while(totalt!=-1){
        print("Please enter number of tickets: ")
        totalt = scanner.nextInt()
        if(totalt==-1){
            println("End");break
        }
        else{
            print("How many round-trip tickets: ")
            roundt = scanner.nextInt()
            val ticket = Ticket(totalt,roundt)
            ticket.print()
        }
    }


}

class Ticket(var totalt: Int, var roundt: Int){
    var total : Int = (roundt * 2000 * 0.9 + (totalt - roundt) * 1000).toInt()
    fun print(){
        println("Total tickets: "+ totalt + "\n" + "Round-trip: "+
                roundt + "\n" + "Total: " + total)
}
}